docs/android/inpath/reservationProcess.md

```diff
@@ -177,6 +177,20 @@ For example if the user's country name code is empty/null on input, this will be
             "@URL": "9c537843-fbdc-4c86-910b-28b159c237c2.64"
         },
         "TPA_Extensions": {
+            "Benefits": {
```

Comment:
Hey @jhowcs, this part is in a wrong place, it should be in the "Customer"->"Primary" object like that:
`"Customer": {
			"Primary": {
				...
				"TPA_Extensions": {
					"Benefits": {
						"Benefit": [{
							"@xmlType": "AV",
							"@type": "CORPORATE_RATE",
							"@code": "A791500"
						}, {
							"@xmlType": "BG",
							"@type": "CORPORATE_RATE",
							"@code": "B204900"
						}]
					}
				}
			}`

Reply:
Thanks for spot this issue Rafa, I'm working on it and I'll have it fixed soon.